src/controllers/article.controller.js

```diff
@@ -124,6 +126,109 @@ class ArticleController {
       `article with id = ${articleId} has been successfully removed from your bookmarks`
     );
   }
+
+  /**
+   *
+   *
+   * @static
+   * @param {*} request -- Request object
+   * @param {*} response -- Response object
+   * @returns {json} -- Returns a json object
+   * @memberof ArticleController
+   */
+  static async createArticle(request, response) {
+    const { id: authorId } = request.currentUser;
+    const { title } = request.body;
+    const article = await BaseRepository.create(db.Article, {
+      ...request.body,
```

Comment:
Nice thought. Considering the making the controller slimmer, this is a good approach because if you put what is not allowed in the request body, it will not pass.